integration-tests/support/pages/ApplicationDetailPage.ts

```diff
@@ -12,10 +13,11 @@ export class ApplicationDetailPage {
     cy.contains('div', applicationDetailPagePO.replicaLabel).should('contain.text', replicaCount);
   }
 
-  checkCpuAndMemory(cpuVal: number, ramValue: number, ramUnit: string) {
+  checkCpuAndMemory(cpuVal: number, cpuUnit: CPUUnit, ramValue: number, ramUnit: MemoryUnit) {
```

Comment:
There is small inconsistency (not caused by you) with `cpuVal` and `ramValue`. Could you please unite them?